src/App.tsx

```diff
@@ -280,6 +319,12 @@ const TeamRound = ({
               </div>
             )
           )}
+          <div className="d-flex flex-row flex-fill">
+            Bench <hr />
```

Comment:
Haha, I tried to put a line to the right to separate the bench from the starters, but don't know exactly how to do that 😅